tools/all-properties-component.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Wonder if we can/should adopt the `esModuleInterop: true` to get rid of this style of import?